md/bonita-docker-installation.md

```diff
@@ -28,45 +28,32 @@ docker run --name=bonita -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:
 First generate a request key into a container with a specific hostname (-h):
 
 ```
-docker run --rm --name=bonita -h bonita -ti quay.io/bonitasoft/bonita-subscription:${varVersion}.0 /bin/bash
-unzip /opt/files/BonitaSubscription-${varVersion}.0.zip 'BonitaSubscription-7.9.0-tomcat/tools/request_key_utils/*'
-cd BonitaSubscription-${varVersion}.0/tools/request_key_utils/
-./generateRequestKey.sh
-exit
+docker run --rm --name=bonita -h bonita -ti quay.io/bonitasoft/bonita-subscription:${varVersion}.0 /bin/bash ./generateRequestKey.sh
 ```
-Stop the docker container.
-Retrieve the licence from the customer portal and place it to a docker volume:
-```
-docker volume create bonita-lic
-docker volume inspect bonita-lic
-```
-This will output the mountpoint which is the host directory asociated to the volume
-```
-[
-    {
-        "CreatedAt": "2019-11-19T09:27:24Z",
-        "Driver": "local",
-        "Labels": {},
-        "Mountpoint": "/var/lib/docker/volumes/bonita-lic/_data",
-        "Name": "bonita-lic",
-        "Options": {},
-        "Scope": "local"
-    }
-]
-```
-Copy the file inside this volume mountpoint
+Answer the questions related to the license you want. This will print out the request key.
+
+Retrieve the licence from the customer portal and place into one folder that will be mounted as volume of the docker image:
+
 ```
-cp ~/Downloads/BonitaSubscription-7.7-Cloud_Techuser-bonita-20170124-20170331.lic ~/var/lib/docker/volumes/bonita-lic/_data
+cp ~/Downloads/BonitaSubscription-7.7-Cloud_Techuser-bonita-20170124-20170331.lic ~/bonita-lic/.
 ```
 
+Alternatively, you can create a named persistent volume in docker for keeping license file. See [docker documentation on volumes](https://docs.docker.com/storage/volumes/).
+
 ### Start the container
 
 Re-create a new Bonita container with the same hostname (-h) and this host directory mounted (-v) :
 
 ```
-docker run --name bonita -h bonita -v bonita-lic:/opt/bonita_lic/ -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:${varVersion}.0
+docker run --name bonita -h bonita -v ~/bonita-lic/:/opt/bonita_lic/ -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:${varVersion}.0
 ```
 
+where `~/bonita-lic/` is the folder containing license file.
+
+::: info
+This will only add the initial license to the Bonita Runtime. To update a license on an existing installation see [Update configuration and license section](#section-update-configuration)
```

Comment:
```suggestion
This will only add the initial license to the Bonita Runtime. To renew a license on an existing installation see [Update configuration and license section](#section-update-configuration)
```